shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
it was already doing this on line [13](https://github.com/rendrjs/rendr/pull/447/files#diff-a11874ca1cacb6a9ac55db82846df825L13)? not sure how it's terribly different from the original functionality there.  Any suggestions to fix it up? my only thought is `this.modelUtils._requireAMD(defaultRouterModule)` instead.